src/main/data_types/dependency_scorer.py

```diff
@@ -93,12 +93,17 @@ def _request_ssf_api(self, dependency: Dependency) -> Dependency:
         Returns:
             Dependency: The dependency with an SSF score.
         """
-        # TODO
-        # 1. Get commit sha1 of the dependency's version
-        # 2. Request the score from the SSF API
-        # 3. If the request is successful construct a new Dependency object
-        # 4. Else return a copy of the old dependency
-        new_dependency = Dependency()
+        sha1 = self._get_git_sha1(dependency.git_url, dependency.version)
+        score = self._lookup_ssf_api(sha1)
```

Comment:
This function does not seem to exist. Is it somewhere else?

Reply:
I think this function is in another branch